integration_tests/rest_api/setup_data_hooks.py

```diff
@@ -132,6 +151,11 @@ def initialize_sample_resources(txns):
     seeded_data['auth_auth'] = auth_account_response['authorization']
     seeded_data['auth_account'] = auth_account_response['account']
 
+    # Create OFFER
+    OFFER['source'] = seeded_data['holding']['id']
+    OFFER['target'] = seeded_data['holding']['id']
```

Comment:
This probably doesn't matter, but do we want to seed two holdings so that source and target don't have to be the same id?